schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
+CREATE DATABASE vet_clinic;
+
 CREATE TABLE animals (
-    name varchar(100)
-);
+    id SERIAL PRIMARY KEY,
```

Comment:
[optional] - you can make your id column not null and autoincremented  by using `    id int NOT NULL IDENTITY(1,1)
`